src/mmw/js/src/draw/views.js

```diff
@@ -172,12 +172,18 @@ var PlaceMarkerView = Marionette.ItemView.extend({
 });
 
 function changeOutlineLayer(endpoint, tableId, model) {
-    var map = App.getLeafletMap();
+    var map = App.getLeafletMap(),
+        ofg = model.get('outlineFeatureGroup');
 
-    if (endpoint && tableId != undefined) {
+    // If the outlines are already present, there is nothing more to do.
+    if (ofg.getLayers().length !== 0) {
```

Comment:
Yeah, that might be a safer option for now until we get some more layers and have a better idea of how that control will work.